packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -153,10 +153,16 @@ export interface IFluidDataStoreChannel extends
     bindToContext(): void;
 
     /**
+     * @deprecated - Replaced by getAttachSummary()
```

Comment:
My preference in such cases is to remove from interface, but leave the implementation.
That has more forceful function on consuming side, while having same level of back-compat.
Also breaking.md change is required no matter what, and // back-compat comment is due in implementation such that we can find it later and remove

Reply:
Though if you are using it in runtime (as a back compat flow), then likely keeping it on interface is fine

Reply:
yeah, that's why I kept it. Added to breaking.md and back-compat comment.